tldw_Server_API/app/core/Ingestion_Media_Processing/Books/Book_Processing_Lib.py

```
@@ -1586,20 +1587,23 @@ def ingest_text_file(file_path, title=None, author=None, keywords=None, base_dir
+            add_media_with_keywords(
+                db_instance=db_instance,
+                url="its_a_book",
+                title=title,
+                media_type='book',
+                content=content,
+                keywords=keywords_list,
+                prompt='No prompt for text files',
+                summary='No summary for text files',
+                transcription_model='None',
+                author=author,
+                ingestion_date=datetime.now().strftime('%Y-%m-%d')
```

Ruff (0.15.6)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)